apps/widclkbttm/widclkbttm.wid.js

```diff
@@ -1,10 +1,12 @@
 WIDGETS["wdclkbttm"]={area:"br",width:Bangle.CLOCK?0:60,draw:function() {
   if (!Bangle.CLOCK == !this.width) { // if we're the wrong size for if we have a clock or not...
-    this.width = Bangle.CLOCK?0:60;
+    this.width = Bangle.CLOCK?0:60;      
     return setTimeout(Bangle.drawWidgets,1); // widget changed size - redraw
   }
   if (!this.width) return; // if size not right, return
-  g.reset().setFont("6x8", 2).setFontAlign(-1, 0).setColor("#0ff"); // cyan
+  if (process.env.BOARD=='BANGLEJS'||process.env.BOARD=='EMSCRIPTEN')  v_font_size=16
```

Comment:
It looks like v_font_size is defined as a global variable here? Please could you define it locally?

Also I'm a bit confused about this to be honest - both Bangles have 24px widget bars, and we're still only leaving 60px for the clock on both - so why do the sizes need to be different?

Also, I'm not sure if a 13pc vector font is really so readable compared to the 16px bitmap font used before?